scripts/emailings/reset-invalid-emails.php

```diff
@@ -67,37 +68,55 @@
 
 $db->begin();
 
+$myfile = fopen($id, "r") or die("Unable to open file\n");
```

Comment:
die must be avoid in Dolibarr dev rules as you can decide the return code. Replace with a test on if and if it fails add a

echo "Failed to open file"
exit(-1)

return code must be < 0 if error.

Also rename $id with $fileofinvalidemail  (a developer must understand what is a var by reading its name).